catkin_ws/src/kpr_testing/src/valves.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/python
```

Comment:
You might want to rename this file as it is not solely used for controlling the valves